config/routes.rb

```diff
@@ -46,7 +46,7 @@
       get 'separation_locations'
     end
 
-    post '/mvi_users/:id', to: 'mvi_users#submit'
+    post '/mvi_users/:id', to: 'mpi_users#submit'
```

Comment:
How about adding a `mpi_users` route as well?

Reply:
I considered this, because we do need a deprecation path.  But I decided not to and let it be part of one of the tickets I need to make for the "These things were not changed"-section of the PR.  Off the top of my head the deprecation work has 3 parts: adding the alternate route, communicating the changeover, and deleting the old route.  My thought was to leave that work to future us, so this PR doesn't get any bigger and make decisions future us might not like.